src/components/Bridge/types.ts

```diff
@@ -1,53 +1,59 @@
 // https://docs.axelar.dev/dev/build/chain-names/mainnet
-export const AXELAR_CHAIN_NAMES = {
-  Avalanche: 'Avalanche',
-  Ethereum: 'Ethereum',
-  Fantom: 'Fantom',
-  Moonbeam: 'Moonbeam',
-  Polygon: 'Polygon',
-  Axelarnet: 'Axelarnet',
-  Cosmoshub: 'cosmoshub',
-  Crescent: 'crescent',
-  Emoney: 'e-money',
-  Injective: 'injective',
-  Juno: 'juno',
-  Kujira: 'kujira',
-  Osmosis: 'osmosis',
-  Secret: 'secret',
-  Terra2: 'terra-2',
-} as const
+import { AccountSpecifier } from 'state/slices/portfolioSlice/portfolioSliceCommon'
 
-export type AxelarChainName = typeof AXELAR_CHAIN_NAMES[keyof typeof AXELAR_CHAIN_NAMES]
+export enum AxelarChainNames {
+  Avalanche = 'Avalanche',
+  Ethereum = 'Ethereum',
+  Fantom = 'Fantom',
+  Moonbeam = 'Moonbeam',
+  Polygon = 'Polygon',
+  Axelarnet = 'Axelarnet',
+  Cosmoshub = 'cosmoshub',
+  Crescent = 'crescent',
+  Emoney = 'e-money',
+  Injective = 'injective',
+  Juno = 'juno',
+  Kujira = 'kujira',
+  Osmosis = 'osmosis',
+  Secret = 'secret',
+  Terra2 = 'terra-2',
+}
+
+export type AxelarChainName = `${AxelarChainNames}`
+
+export const isAxelarChainName = (chainName: any): chainName is AxelarChainName => {
+  return Object.values(AxelarChainNames).includes(chainName as AxelarChainNames)
+}
```

Comment:
Since `name` vars are properly typed as `AxelarChainName`, there is no use for this and we can delete it? https://github.com/shapeshift/web/pull/2180#discussion_r939596955

Reply:
You're right, we didn't need to cast, but we actually don't need `isAxelarChainName` at all now that we've improved the types.

Removed in https://github.com/shapeshift/web/pull/2353/commits/5c3ca8ab22726d06d61968dcc496d42db04f5edb